### ISSUE TITLE
Incorrectly removing space

### ISSUE THREAD

**Issue by Mottie** [jakubpawlowicz/clean-css#735]
If I compress the following css:

```css
.highlight-code-c\+\+ .keyword { color:red; }
```

the result is:

```css
.highlight-code-c\+\+.keyword{color:red}
```

The space after the second `+` should not be removed.

This result is seen when using http://refresh-sf.com/ and node-cssmin v0.14.0. My OS is Windows 10 64bit.

**Comment by silverwind** [comment]
Slightly reduced case:

```console
$ echo ".c\+\+ .a {color: red}" | cleancss
.c\+\+.a{color:red}
$ echo ".c\+\+ .a {color: red}" | csso
.c\+\+ .a{color:red}
$ cleancss -v
3.4.9
```

**Comment by silverwind** [comment]
Filed #736.

**Comment by jakubpawlowicz** [comment]
Released as 3.4.10.
